### ISSUE TITLE
Execution failed for task ':marytts-languages:marytts-lang-fr:test'.

### ISSUE THREAD

**Issue by elnursamedzade** [marytts/marytts#988]
Hello there.
I am interested in tr language. Does the error below have anything to do with me?

```
23 tests completed, 6 failed

> Task :marytts-languages:marytts-lang-fr:test FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':marytts-languages:marytts-lang-fr:test'.
> There were failing tests. See the report at: file:///C:/Users/Elnur/marytts/marytts-languages/marytts-lang-fr/build/reports/tests/test/index.html

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BU¦LD FAILED in 5m 55s
146 actionable tasks: 146 executed

C:\Users\Elnur\marytts> 
```

**Comment by seblemaguer** [comment]
Are you on windows? What do you see in file:///C:/Users/Elnur/marytts/marytts-languages/marytts-lang-fr/build/reports/tests/test/index.html ?

**Comment by seblemaguer** [comment]
It seems to be related to #987

**Comment by elnursamedzade** [comment]
Yes, I am using windows10.
In the index.html file it says:

Test Summary
23
tests
6
failures
0
ignored
0.474s
duration
73%
successful
Failed tests
Packages
Classes
PreprocessTest. testExpandOrdinal[0](2, deuxiÃ¨me)
PreprocessTest. testExpandOrdinal[1](3, troisiÃ¨me)
PreprocessTest. testExpandOrdinal[2](4, quatriÃ¨me)
PreprocessTest. testSpellout[7](2e, deuxiÃ¨me)
PreprocessTest. testSpellout[8](3e, troisiÃ¨me)
PreprocessTest. testSpellout[9](4e, quatriÃ¨me)
Generated by Gradle 6.3 at 19.May.2020 19:14:39

Sébastien Le Maguer <notifications@github.com>, 19 May 2020 Sal, 19:41
tarihinde şunu yazdı:

**Comment by seblemaguer** [comment]
You are facing the problem of issue #987, we are working on it

**Comment by elnursamedzade** [comment]
Yes I understood. Thank you so much. Good work.

Sébastien Le Maguer <notifications@github.com>, 19 May 2020 Sal, 20:36
tarihinde şunu yazdı:

**Comment by psibre** [comment]
Solved via #989
